qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/hover/QuteHoverInExpressionWithLetSectionTest.java

```diff
@@ -55,4 +55,28 @@ public void assignedString() throws Exception {
 				"```", //
 				r(0, 10, 0, 18));
 	}
+
+	@Test
+	public void hoverMethodPart() throws Exception {
+		String template = "{@java.lang.String string}\r\n" + //
+				"{#let name=string.cha|rAt(0) /}";
+		assertHover(template, "```java" + //
+				System.lineSeparator() + //
+				"char java.lang.String.charAt(int index)" + //
+				System.lineSeparator() + //
+				"```", //
+				r(1, 18, 1, 24));
+	}
+	
+	@Test
+	public void hoverParameterOfMethodPart() throws Exception {
+		String template = "{@java.lang.String string}\r\n" + //
+				"{#let name=string.charAt(0|) /}";
+		assertHover(template, "```java" + //
+				System.lineSeparator() + //
+				"Integer" + //

```

Comment:
It would be nice to have `int` here if possible, I will create a bug for that if there isn't already one.

Reply:
#720